sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Release History
+## 12.0.0 (2019-10-31)
```

Comment:
2019-10-29

Reply:
Is this change only for identity CHANGELOG or for every other CHANGELOG ?